packages/panna-sdk/src/react/utils/query-utils.ts

```diff
@@ -0,0 +1,26 @@
+const MAX_LIMIT = 100;
+const MAX_OFFSET = 1000;
```

Comment:
I think, we should not set a MAX_OFFSET here. Technically, a user can own NFTs from more than 1000 collections.

Reply:
I'd recommend that we remove `MAX_OFFSET`. `getCollectiblesByAddress` will return and empty array if no eligible data exists.